### PR TITLE
updated German translations

### PR DESCRIPTION
**Pull request by plata**

### CODE REVIEW

**Comment by plata** [comment]
@madoar can you review this? Once this is merged, I can update the translations for EA Origin and finish that PR as well.

File: i18n/Messages_de.properties [hunk 1]
```diff
@@ -4,429 +4,561 @@
...
+Crayon\ Physics=Crayon Physics
 
 #: i18n/json_string.js:28
-<b>Introduction</b><br/>PhotoFiltre\ Studio\ is\ a\ complete\ image\ retouching\ program.\ It\ allows\ you\ to\ do\ simple\ or\ advanced\ adjustments\ to\ an\ image\ and\ apply\ a\ vast\ range\ of\ filters\ on\ it.\ It\ is\ simple\ and\ intuitive\ to\ use,\ and\ has\ an\ easy\ learning\ curve.\ The\ toolbar,\ giving\ you\ access\ to\ the\ standard\ filters\ with\ just\ a\ few\ clicks,\ gives\ PhotoFiltre\ Studio\ a\ robust\ look.\ PhotoFiltre\ Studio\ also\ has\ layer\ manager\ (with\ Alpha\ channel),\ advanced\ brushes,\ nozzles\ (or\ tubes),\ red\ eye\ corrector,\ batch\ module\ and\ lot\ of\ other\ powerful\ tools.=PhotoFiltre ist ein Bildbearbeitungsprogramm, welches Ebenen, Filter und Effekte unterst\u00fctzt.
+Crayon\ Physics\ is\ a\ mouse\ arcade\ game.\ You\ will\ have\ to\ draw\ lines\ and\ squares\ to\ move\ a\ ball.\ The\ aim\ is\ to\ catch\ the\ stars\ in\ the\ level.\ =Crayon Physics ist ein Arcade Spiel, bei dem Sie mit der Maus Linien und Vierecke zeichnen m\u00fcssen, um mit einem Ball den Stern am Ende des Levels zu erreichen.
 
 #: i18n/json_string.js:29
-Icy\ Tower\ 1.5=Icy Tower 1.5
+Custom=Eigene
```
Comment (madoar) [inline i18n/Messages_de.properties]: I'm not sure `Eigene` is a good translation here, maybe `Angepasst` is better?

File: i18n/Messages_de.properties [hunk 2]
```diff
@@ -4,429 +4,561 @@
...
-Other=Nicht zuzuordnen
+Relive\ Half-Life\ in\ this\ highly\ acclaimed,\ fan-made\ recreation=Erleben Sie Half-Life neu in dieser vielgepriesenen Neugestaltung durch Fans.
 
 #: i18n/json_string.js:96
-Functions=Functions
+Resident\ Evil\ 3=Resident Evil 3
 
 #: i18n/json_string.js:97
-League\ of\ Legends\ BETA\ Client=League of Legends BETA Client
+Resident\ Evil\ 3\:\ Nemesis,\ known\ in\ Japan\ as\ Biohazard\ 3\:\ Last\ Escape\ (\u30d0\u30a4\u30aa\u30cf\u30b6\u30fc\u30c93\u3000\u30e9\u30b9\u30c8\u30a8\u30b9\u30b1\u30fc\u3
```
Comment (madoar) [inline i18n/Messages_de.properties]: Are your sure `Science` equals `Lernprogramme`? I think `Wissenschaft` fits better here

Comment (plata) [inline i18n/Messages_de.properties]: Yes, it's correct. I used the translations from Kubuntu 16.04.

**Pull request merged**